src/main/java/jenkins/scm/api/SCMFileSystem.java

```diff
@@ -187,15 +187,37 @@ public static SCMFileSystem of(@NonNull Item owner, @NonNull SCM scm) throws IOE
      * @throws InterruptedException if the attempt to create a {@link SCMFileSystem} was interrupted.
      */
     @CheckForNull
-    public static SCMFileSystem of(@NonNull Item owner, @NonNull SCM scm, @CheckForNull SCMRevision rev)
+    public static SCMFileSystem of(@NonNull Item owner, @NonNull SCM scm,@CheckForNull SCMRevision rev) throws IOException, InterruptedException {
+        return of(owner, scm, rev,null);
+    }
+
+
+    /**
+     * Given a {@link SCM} this method will try to retrieve a corresponding {@link SCMFileSystem} instance that
+     * reflects the content at the specified {@link SCMRevision}.
+     *
+     * @param owner the owner of the {@link SCM}
+     * @param scm the {@link SCM}.
+     * @param rev the specified {@link SCMRevision}.
+     * @param build the specified {@link Run}.
```

Comment:
None of these tags actually tell the reader anything. The existing ones are not your doing, but while we are adding new ones, suggest perhaps
```suggestion
     * @param build an associated build context, if any, that could be used for example to look up parameters
```